gsy_framework/sim_results/electric_blue/all_results.py

```diff
@@ -147,10 +147,13 @@ def _generate_asset_volume_time_series(
 
         asset_uuid = current_asset_stats.device_uuid
         if asset_uuid not in self.asset_volume_time_series:
+            asset_peak_kwh = (
+                asset_info["capacity_kW"] if "capacity_kW" in asset_info
+                else asset_info["avg_power_W"])
```

Comment:
I do not remember this part of the code, but the units look wrong. Here W or kW values are assigned to a kWh value?

Reply:
There is a reason why you do not remember this part of the code, it is for Electric Blue :) I added a comment to signify that slot length is hourly in this case, and converted the units from W to kWh. Thanks for the remark.

Reply:
Thanks! And the variable is ok to be named "asset_peak_kwh" because this is a hourly resolution?

Reply:
Yes, I left it as an extra comment even, in order to denote that no power -> energy conversion is needed here.